### ISSUE TITLE
Add ngx-translate

### ISSUE THREAD

**Issue by sulco** [sulco/angular-developer-roadmap#2]

**Comment by skix123** [comment]
Please use [Angular i18n](https://angular.io/guide/i18n) instead.

**Comment by sulco** [comment]
@skix123 Yeah, makes sense to add it, but I think I'll mark it is a possibility, since it lacks dynamically loaded translations which is one of main features most devs I've been talking to are looking for in translation solutions (ngx-translate supports that)
